src/main/java/dao/ProductAccountDao.java

```diff
@@ -0,0 +1,15 @@
+package dao;
+
+import java.util.List;
+import model.ProductAccount;
+
+public interface ProductAccountDao {
+
+    public void add(ProductAccount product);
+
+    public ProductAccount get(String productName);
+
+    public void update(ProductAccount product);
+
+    public List<ProductAccount> getBalance();
```

Comment:
```suggestion
    void add(ProductAccount product);

    ProductAccount get(String productName);

    void update(ProductAccount product);

    List<ProductAccount> getBalance();
```

they are public by default